frontend/server/src/DAO/Assignments.php

```diff
@@ -56,6 +56,9 @@ public static function getAssignmentsProblemsStatistics(
             bpr.problem_alias,
             VARIANCE(bpr.max_user_score_for_problem) AS variance,
             AVG(bpr.max_user_score_for_problem) AS average,
+            AVG(
+                CASE WHEN bpr.max_user_percent_for_problem > 0.6 THEN 1 ELSE 0 END
```

Comment:
aquí esto no debería ser

```suggestion
                CASE WHEN bpr.max_user_percent_for_problem = 1 THEN 1 ELSE 0 END
```

? se puede agregar una prueba con dos usuarios, uno que resuelva 100% y otro que resuelva 90% y que sólo el primero se encuentre en esta categoría?

Reply:
Podría ser cambiando `CASE WHEN bpr.max_user_percent_for_problem > 0.6 THEN 1 ELSE 0 END` por `CASE WHEN bpr.max_user_percent_for_problem > 0.6 AND bpr.max_user_percent_for_problem < 1 THEN 1 ELSE 0 END` ?

Igual puedo agregar a prueba si es necesario.

Reply:
hmm pero eso igual incluiría gente que resolvió menos del 100%, no?

sí hay que agregar la prueba, porque ese `0.6` no concuerda mucho con el "100%" de la descripción :S